tests/test_full.nf.test

```diff
@@ -251,6 +251,73 @@ nextflow_pipeline {
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664516.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664517.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664518.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664520.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664521.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664523.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664524.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/abricate/argnorm/ERZ1664528.normalized.tsv"),
+                ).match("argnorm_abricate") },
+
+                { assert snapshot (
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664501.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664503.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664504.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664505.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664506.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664507.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664508.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664509.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664510.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664511.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664515.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664516.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664517.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664518.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664520.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664521.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664523.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664524.normalized.tsv"),
+                    path("$outputDir/arg/hamronization/amrfinderplus/argnorm/ERZ1664528.normalized.tsv"),
+                ).match("argnorm_amrfinderplus") },
```

Comment:
@jasmezz you should generate the snapshot when the PR is otherwise ready for merged, due to the computiational requirements